src/custom/pages/Claim/ClaimsOnOtherChainsBanner.tsx

```diff
@@ -0,0 +1,84 @@
+import { useMemo } from 'react'
+import styled from 'styled-components/macro'
+import { PhishAlert } from 'components/Header/URLWarning'
+import { NETWORK_LABELS, SupportedChainId } from 'constants/chains'
+import { useClaimState } from 'state/claim/hooks'
+import useChangeNetworks from 'hooks/useChangeNetworks'
+
+const ChainSpan = styled.span``
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: auto;
+  flex-flow: row wrap;
+
+  > div {
+    flex: 1 1 auto;
+  }
+
+  > div:nth-child(2) {
+    > span {
+      margin-left: 4px;
+      font-weight: 600;
+      white-space: nowrap;
+      cursor: pointer;
+      text-decoration: underline;
+
+      &:last-child {
+        &:after {
+          content: '!';
+        }
+      }
+
+      &:not(:last-child) {
+        &:after {
+          content: ',';
+        }
+      }
+    }
+  }
+`
+
+function ClaimsOnOtherChainsBanner({ className }: { className?: string }) {
+  const { callback } = useChangeNetworks()
+
+  const { hasClaimsOnOtherChains } = useClaimState()
+  const chainsWithClaims: SupportedChainId[] = useMemo(
+    () =>
+      Object.keys(hasClaimsOnOtherChains).reduce((acc, chain) => {
```

Comment:
can you skip the current network?

Also, the reduce could also return a flag saying if the current one has it?

With these 2 pieces of info, we can do the message i was suggesting.

Reply:
shouldnt even be showing the chain we're on, i checked this. likely i messed sth up when testing. updater checks current chain

Reply:
fixed